src/polygon.cpp

```diff
@@ -528,7 +527,7 @@ class EarClip {
 
     void PrintVert() const {
 #ifdef MANIFOLD_DEBUG
-      if (!params.verbose) return;
+      if (!ManifoldParams().verbose) return;
```

Comment:
We'll need to separate the verbosity or have levels or something. The triangulator is orders of magnitude more verbose than the Boolean is.

Reply:
it is now an int